src/test/java/uk/gov/justice/hmpps/prison/repository/InmateRepositoryTest.java

```diff
@@ -398,9 +398,7 @@ public void testFindOffendersWithDateOfBirthRange() {
 
         final var offenders = findOffenders(query);
 
-        assertThat(offenders).hasSize(9);
-        assertThat(offenders).extracting(PrisonerDetail::getOffenderNo)
-                .contains("A1234AA", "A1234AF", "A1234AL", "Z0019ZZ", "Z0020ZZ", "Z0021ZZ", "Z0022ZZ", "Z0023ZZ", "A1180MA");
```

Comment:
this line was fine, it was using `contains` so extra items would be allowed

Reply:
duh

Reply:
duh - updated